_pages/projects.md

```diff
@@ -44,8 +44,7 @@ Template:
 
 
 ![team-10.png](/assets/images/projects/team-10.png){:class="img-final-projects"}
-* Team 10 Project
-
+* [**SDFGI: Real-time SDF-based Global Illumination**](https://github.com/carlos-lopez-garces/SDFGI) -- [Michael Mason](https://github.com/micklemacklemore), [Carlos Lopez Garces](https://github.com/carlos-lopez-garces), [Logan Cho](https://github.com/logancho)
```

Comment:
Are you also planning to add an image? Its fine if you want to update the image later, you can open a new PR.